proxy/http2/Http2ConnectionState.cc

```diff
@@ -1146,6 +1163,11 @@ Http2ConnectionState::restart_streams()
 void
 Http2ConnectionState::cleanup_streams()
 {
+  if (post_stream_count > client_streams_in_count) {
+    HTTP2_SUM_THREAD_DYN_STAT(HTTP2_STAT_TOTAL_CLIENT_STREAM_POST_GOAWAY, this->mutex->thread_holding,
```

Comment:
The name of `post_stream_count` may be a bit confusing because `post_stream_count - client_streams_in_count` is the actual number of it.